Packages/Netherlands3D/Timeline/Runtime/Scripts/TimeBarInteraction.cs

```diff
@@ -0,0 +1,92 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace Netherlands3D.Timeline
+{
+    public class TimeBarInteraction : MonoBehaviour, IPointerDownHandler, IPointerUpHandler, IPointerEnterHandler, IPointerExitHandler
+    {
+        [SerializeField] private float sensitivity = 1000;
+
+        [Header("Components")]
+        [Tooltip("The timeline UI this interaction is part of")]
+        public TimelineUI timelineUI;
+
+        /// <summary>
+        /// Is the user dragging its mouse on the time bar?
+        /// </summary>
+        private bool isDragging;
+        /// <summary>
+        /// Is the mouse on the ui?
+        /// </summary>
+        private bool mouseIsOn;
+        /// <summary>
+        /// The position of the mouse when left input was pressed
+        /// </summary>
+        private Vector3 mouseDownPosition;
+
+        // Update is called once per frame
+        void Update()
+        {
+            OnPointerStay();
+        }
+
+        /// <summary>
+        /// User presses mouse down on time bar
+        /// </summary>
+        /// <param name="eventData"></param>
+        public void OnPointerDown(PointerEventData eventData)
+        {
+            isDragging = true;
+            mouseDownPosition = Input.mousePosition;
+            print("yo");
```

Comment:
eventData heeft een position (eventData.position), die kan je beter gebruiken dan Input.mousePosition. Dan is het EventSystem degene die bepaald wat het pointer device is

Reply:
Dit was nog een oud script dat niet meer gebruikt werd dus heb het verwijderd